examples/api-tests/src/file-search.spec.js

```diff
@@ -55,6 +57,59 @@ describe('file-search', function () {
 
         });
 
+        describe('#buildOpenerOptions', () => {
+            it('should return the range', () => {
+                const rangeTest = {
+                    start: { line: 3, character: 3 },
+                    end: { line: 5, character: 5 }
+                };
+                quickFileOpenService['filterAndRange'].range = rangeTest;
+                expect(quickFileOpenService['buildOpenerOptions']().selection).eq(rangeTest);
+            });
```

Comment:
I don't see how this test will pass?
If no search is performed, then the `filterAndRange` should be the default:

```ts
protected filterAndRangeDefault = { filter: '', range: undefined };
```